functions-networking-utils.html

```diff
@@ -181,8 +180,8 @@ <h4>
 		<li>This always return the most up-to-date information we have available right now, even if we are in the middle of re-calculating ping times.</li>
 		<li>Returns a <strong>dictionary</strong>:
 			<ul class="inner">
-				<li>location (int)</li>
-				<li>ping (float)</li>
+				<li>location (PoolByteArray)</li>
+				<li>age of the data (float). You can use this value in CheckPingDataUpToDate</li>
```

Comment:
This line should just be 'age' as these show the names of the keys for the returned dictionary.  Any explanation of what they are should be in the list above the "return" line.  Like...

<li>Age of the data is returned as 'age'; you can use this value in <strong>checkPingDataUpToDate</strong>.</li>
<li>Returns a <strong>dictionary</strong>:
    <ul class="inner">
        <li>location (PoolByteArray)</li>
        <li>age (float)</li>
    </ul>
</li>

That or we change the key name to age_of_data, which would be more descriptive and better really.

Reply:
Yes, it will be more consistent.
I changed it. don't hesitate to change if you don't like the formulation ;)